src/postgres/registerPostgresCommands.ts

```diff
@@ -0,0 +1,17 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { IActionContext, registerCommand } from "vscode-azureextensionui";
+import { ext } from '../extensionVariables';
+import { PostgresDatabaseTreeItem } from "./tree/PostgresDatabaseTreeItem";
+
+export function registerPostgresCommands(): void {
+    registerCommand('cosmosDB.deletePostgresDatabase', async (context: IActionContext, node?: PostgresDatabaseTreeItem) => {
```

Comment:
I expect this file will grow pretty fast with time. Please structure it like here: https://github.com/microsoft/vscode-azurefunctions/blob/master/src/commands/registerCommands.ts

Two highlights:
1. It's in a folder called "commands"
1. Each call to `registerCommand` is a single line of code